### DISCUSSION TITLE
cli completion for fig

### DISCUSSION THREAD

**Discussion by landsman** [supabase/supabase#18497] [Feature Requests]
I noticed that you can use command in CLI `supabase completion zsh`:

```
Generate the autocompletion script for supabase for the specified shell.
See each sub-command's help for details on how to use the generated script.

Usage:
  supabase completion [command]

Available Commands:
  bash        Generate the autocompletion script for bash
  fish        Generate the autocompletion script for fish
  powershell  Generate the autocompletion script for powershell
  zsh         Generate the autocompletion script for zsh

Flags:
  -h, --help   help for completion

Global Flags:
      --debug                             output debug logs to stderr
      --dns-resolver [ native | https ]   lookup domain names using the specified resolver (default native)
      --experimental                      enable experimental features
      --workdir string                    path to a Supabase project directory

Use "supabase completion [command] --help" for more information about a command.
```

but I don't know from this doc how to make it workf with Fig.
What about to write official plugin for Fig which I can install in their s app?

![Screenshot 2023-10-26 at 13 54 16](https://github.com/supabase/supabase/assets/3184228/1bec74b6-560d-4507-a4bd-5b2d2dff28ed)

https://fig.io/user-manual/plugins

**Comment by encima** [comment]
You can add the completion to your .zshrc by adding `eval "$(supabase completions zsh)"` to the end of the file.

Of course, you are welcome to create a community plugin using the completions if you like.

**Reply by richard-edwards** [reply]
Slight typo there. Should be `eval "$(supabase completion zsh)"`. (unplural) Thanks for the help!
